Resources/config/managers.yml

```diff
@@ -4,14 +4,15 @@ parameters:
 
 services:
     pim_custom_entity.manager.registry:
-        class: %pim_custom_entity.manager.registry.class%
+        class: '%pim_custom_entity.manager.registry.class%'
 
     pim_custom_entity.manager:
-        class: %pim_custom_entity.manager.class%
+        class: '%pim_custom_entity.manager.class%'
         arguments:
             - '@doctrine.orm.entity_manager'
             - '@pim_custom_entity.updater.custom_entity'
             - '@pim_custom_entity.saver.custom_entity'
             - '@pim_custom_entity.remover.custom_entity'
+            - '@pim_custom_entity.normalizer.internal_api.custom_entity'
```

Comment:
I think you should inject the serializer. Then it's up to the serializer to know which normalizer should be called